.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  release:
+    types: [released]
+
+name: Upload Gutenberg plugin to WordPress.org plugin repo
+
+jobs:
+  upload:
+    name: Upload Gutenberg Plugin
+    runs-on: ubuntu-latest
+    if: github.event.release.assets[0]
+    env:
+      PLUGIN_REPO_URL: 'https://plugins.svn.wordpress.org/gutenberg'
+      STABLE_TAG_REGEX: 'Stable tag: [0-9]\+\.[0-9]\+\.[0-9]\+\s*'
+      SVN_USERNAME: ${{ secrets.svn_username }}
+      SVN_PASSWORD: ${{ secrets.svn_password }}
+      VERSION: ${{ github.event.release.name }}
+    steps:
+      - name: Check out Gutenberg trunk from WP.org plugin repo
+        run: svn checkout "$PLUGIN_REPO_URL/trunk"
+
+      - name: Get previous stable tag
+        id: get_previous_stable_tag
+        run: echo ::set-output name=stable_tag::$(grep "$STABLE_TAG_REGEX" ./trunk/readme.txt)
+
+      - name: Delete everything
+        working-directory: ./trunk
+        run: find . -maxdepth 1 -not -name ".svn" -not -name "." -not -name ".." -exec rm -rf {} +
+
+      - name: Download and unzip Gutenberg plugin asset into trunk folder
+        env:
+          PLUGIN_URL: ${{ github.event.release.assets[0].browser_download_url }}
+        run: |
+          curl -L -o gutenberg.zip $PLUGIN_URL
+          unzip gutenberg.zip -d trunk
+          rm gutenberg.zip
+
+      - name: Replace the stable tag placeholder with the existing stable tag on the SVN repository
+        env:
+          STABLE_TAG_PLACEHOLDER: 'Stable tag: [0-9]\+\.[0-9]\+\.[0-9]\+\s*'
```

Comment:
Is this different from `STABLE_TAG_REGEX`, already present in the broader environment of the `'upload'` job?

Reply:
Ooh, good spot! Looks like I made a mistake when carrying over this logic from `bin/plugin/commands/release.js`. I'll fix!

https://github.com/WordPress/gutenberg/blob/e892c21474638de9c8f749adc34c5c3cc5c8e9b4/bin/plugin/commands/release.js#L35-L36